backend/src/test/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.jpa.hibernate.ddl-auto=create
```

Comment:
Warum hier create und in der main application.properties none?

Reply:
Ist das in der Testumgebung nicht eine in memory Datenbank und deshalb ist es hier in Ordnung ? Aber bei der Datenbank die nachher produktiv geht möchte man nicht das die das einfach so übernimmt bei Änderungen weil da Probleme entstehen können die man nicht möchte? Ich habe keine Ahnung aber kann mir vorstellen das Florian sich das so gedacht haben könnte ?

Reply:
Ist ja keine in Memory, wird ja extra ein testcontainer mit PostgreSQL hochgefahren über docker

Reply:
Aha ok so läuft das :D

Reply:
Ja ich würde nochmal überlegen ob wir wirklich TestContainers nutzen wollen... 
Aber ja ich glaube auto-ddl none sollte genau so gehen, dafür haben wir ja flyway nicht das er ein anderes schema generiert als wir eigentlich haben wollten